include/aspect/material_model/melt_simple.h

```diff
@@ -49,8 +50,9 @@ namespace aspect
      * Interface::is_compressible) only if this is specified in the input file,
      * and contains compressibility for both solid and melt.
      *
-     * @ingroup MaterialModels
+     * @ingroup MeltModel
      */
+    namespace MeltModel {}
```

Comment:
I dont understand this change. The class `MeltSimple` is still a material model, not a melt model, so its documentation should stay in the `MaterialModels` group.

And opening and closing the namespace `MeltModel` will do nothing here. Please remove.